controllers/home.js

```diff
@@ -4,7 +4,9 @@ const AppConstants = require("../app-constants");
 const DB = require("../db/DB");
 const HIBP = require("../hibp");
 const { scanResult } = require("../scan-results");
-const { generatePageToken } = require("./utils");
+const { generatePageToken, getExperimentBranch, getExperimentFlags } = require("./utils");
+
+const EXPERIMENTS_ENABLED = (AppConstants.EXPERIMENT_ACTIVE === "1");
```

Comment:
Just to cut down on code churn from future experiments ... we could push a lot of this logic up into some kind of `ExperimentUtils.init()` module + call. That way the linter will like it but we won't have to keep adding and removing lines like this.

Reply:
And then `ExperimentUtils.getExperimentBranch()`, `ExperimentUtils.getExperimentFlags()`, etc.

Reply:
Filed! https://github.com/mozilla/blurts-server/issues/1738